### ISSUE TITLE
`Point.Empty` and `Point.IsEmpty` make no sense

### ISSUE THREAD

**Issue by tig** [gui-cs/Terminal.Gui#2902]
a) The words in the code below make no sense and are in conflict with each other. Does "Empty" mean "uninitialized" or `0,0`?
b) In the face of `nullable` does it even make sense to try to have such a construct?

```cs
/// <summary>
///	Empty Shared Field
/// </summary>
///
/// <remarks>
///	An uninitialized Point Structure.
/// </remarks>

public static readonly Point Empty;

/// <summary>
///	IsEmpty Property
/// </summary>
///
/// <remarks>
///	Indicates if both X and Y are zero.
/// </remarks>		
[System.Text.Json.Serialization.JsonIgnore]
public bool IsEmpty {
	get {
		return ((X == 0) && (Y == 0));
	}
}
```

**Comment by tig** [comment]
Fixed by @dodexahedron in #3268. Yay!

**Comment by dodexahedron** [comment]
Well, kinda. 😅

More of a "wontfix" because those are in the System.Drawing types, too. I'm sure that's why they were in our copies, anyway, since they had initially been copied from earlier versions of the System.Drawing types.

I agree the names are unfortunate, on the Point type, but it's actually a common term for (0,0), and not just in dotnet either.

I suppose it would allow a fair amount of code reuse between Point, Size, Rectangle, and other drawing primitives. In C++ or C#, you could use templates or generics to handle the all-zeros value if the same property exists on all of them. But is that the reason? Who knows. It's been that way as long as I can remember. 🤷‍♂️
